test/item_test.rb

```diff
@@ -41,10 +41,45 @@ def test_find_merchant_returns_a_merchant_associated_with_item
     assert_equal 'Schroeder-Jerde', result_1.name
   end
 
+  def test_best_day_returns_date_with_the_most_sales_for_given_item_using_the_invoice_date
+    load_data
+    result_1 = @se.item_repository.items[0].best_day
+
+    assert_equal Date.parse("Jun 30, 2015"), result_1
+  end
   private
 
   def fixtures_directory
-    File.expand_path('../../data/fixtures', __FILE__)
+    File.expand_path('../../data/fixtures',__FILE__)
+  end
+
+  def load_data
+    repo_data = {
+        items: [
+            { id: 1, unit_price: 5_000, created_at: "Jan, 15, 2015", updated_at: "Jan, 15, 2015" },
+        ],
+
+        invoices: [
+            { id: 10, created_at: "June 1, 2015" },
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.